### PR TITLE
13.0: s_s_available_to_promise_release: fixes

### PR DESCRIPTION
**Pull request by mmequignon**
Fix availability status which was computed on sections and delivery lines.

### CODE REVIEW

File: sale_stock_available_to_promise_release/models/sale_order_line.py [hunk 1]
```diff
@@ -30,6 +30,16 @@ class SaleOrderLine(models.Model):
     )
     def _compute_availability_status(self):
         for record in self:
+            if record.display_type:
+                record.availability_status = False
+                record.expected_availability_date = False
+                record.available_qty = False
+                continue
+            if record.is_delivery:
```
Comment (simahawk) [inline sale_stock_available_to_promise_release/models/sale_order_line.py]:
```suggestion
            elif record.is_delivery:
```

**Review by simahawk — APPROVED** [review]

File: sale_stock_available_to_promise_release/models/sale_order_line.py [hunk 2]
```diff
@@ -23,13 +23,31 @@ class SaleOrderLine(models.Model):
...
         "product_uom_qty",
     )
     def _compute_availability_status(self):
         for record in self:
+            if record.display_type or not record.product_id:
+                record.availability_status = False
+                record.expected_availability_date = False
+                record.available_qty = False
+                continue
+            elif record.is_delivery:
```
Comment (sebalix) [inline sale_stock_available_to_promise_release/models/sale_order_line.py:48]: `is_delivery` and `display_type` should be added in `api.depends` to be consistent (they are not going to change over time that said)

Comment (simahawk) [reply sale_stock_available_to_promise_release/models/sale_order_line.py:48]: @mmequignon ping. Also, squash when done pls

Comment (simahawk) [reply sale_stock_available_to_promise_release/models/sale_order_line.py:48]: @mmequignon ping

File: sale_stock_available_to_promise_release/i18n/fr_CH.po [hunk 3]
```diff
@@ -6,7 +6,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 13.0+e\n"
```
Comment (TDu) [inline sale_stock_available_to_promise_release/i18n/fr_CH.po:7]: I really think this file should be renamed fr.po so it is useful for every French speaking localisation.
@mmequignon

Comment (mmequignon) [reply sale_stock_available_to_promise_release/i18n/fr_CH.po:7]: Done

**Comment by sebalix** [comment]
@mmequignon can you squash please?

**Comment by mmequignon** [comment]
@sebalix, @TDu could you please make another pass?

**Review by sebalix — APPROVED** [review]

**Pull request merged**